panels/exo_skel.py

```diff
@@ -81,6 +81,8 @@ class BuildBoneList(bpy.types.Operator):
     bl_options = {'REGISTER', 'UNDO', 'INTERNAL'}
     
     def execute(self, context):
+        bpy.ops.wm.save_mainfile()
```

Comment:
This prevents building the bone list if there are any warnings while saving like missing textures. We shouldn't need this line.

Reply:
OK. So the reason why mine wasn't rebuilding was due to missing textures?

Reply:
I mean that calling `bpy.ops.wm.save_mainfile()` can fail. I had texture nodes referring to image files that didn't exist on disk. The bone list updates just fine for me without saving the Blender file. Are you able to reproduce the issue you mentioned where you needed to call `bpy.ops.wm.save_mainfile()`?

Reply:
https://youtu.be/X1fPgppsb6Y